client/my-sites/sidebar/sidebar.jsx

```diff
@@ -447,7 +447,7 @@ export class MySitesSidebar extends Component {
 				link={ siteSettingsLink }
 				onNavigate={ this.onNavigate }
 				icon="cog"
-				preloadSectionName="settings"
+				preloadSectionName="settings-general"
```

Comment:
>  it'll load anyway every time when we load any of the settings sections anyway

This is the sidebar link, so `settings` may not yet be loaded. The performance regression is that on master, a hover over the sidebar link preloads everything necessary to display the general settings page. In this PR, we have split the section into two and are only preloading one of them.